docs/source/examples/smoothing.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<p>Quick example here? Maybe one of these last two can use a hindcast setup to show there's no special differences here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>added</p>